test/tangy-form_test.html

```diff
@@ -616,6 +628,103 @@
+              "linearMode": true,
+              "hideClosedItems": true,
+              "hideCompleteFab": false,
+              "tabIndex": 0,
+              "showResponse": false,
+              "showSummary": false,
+              "hasSummary": false,
+              "onSubmit": "",
+              "id": "test",
+              "tagName": "TANGY-FORM"
+            },
+            "items": [
+              {
+                "id": "item-1",
+                "src": "tangy-form-item",
+                "title": "",
+                "summary": false,
+                "hideButtons": true,
+                "hideBackButton": false,
+                "rightToLeft": false,
+                "hideNextButton": false,
+                "showCompleteButton": false,
+                "inputs": [],
+                "open": false,
+                "incomplete": true,
+                "disabled": false,
+                "hidden": true,
+                "locked": false,
+                "tagName": "TANGY-FORM-ITEM"
+              },
+              {
+                "id": "item-2",
+                "src": "tangy-form-item",
+                "title": "",
+                "summary": false,
+                "hideButtons": true,
+                "hideBackButton": true,
+                "rightToLeft": false,
+                "hideNextButton": false,
+                "showCompleteButton": false,
+                "inputs": [],
+                "open": false,
+                "incomplete": true,
+                "disabled": false,
+                "hidden": false,
+                "locked": false,
+                "tagName": "TANGY-FORM-ITEM"
+              },
+              {
+                "id": "item-3",
+                "src": "tangy-form-item",
+                "title": "",
+                "summary": false,
+                "hideButtons": true,
+                "hideBackButton": true,
+                "rightToLeft": false,
+                "hideNextButton": false,
+                "showCompleteButton": false,
+                "inputs": [],
+                "open": false,
+                "incomplete": true,
+                "disabled": false,
+                "hidden": false,
+                "locked": false,
+                "tagName": "TANGY-FORM-ITEM"
+              }
+            ],
+            "inputs": [],
+            "complete": false,
+            "focusIndex": 0,
+            "nextFocusIndex": 1,
+            "previousFocusIndex": -1,
+            "startDatetime": "7/17/2018, 6:13:38 PM",
+            "startUnixtime": 1531865618651,
+            "uploadDatetime": ""
+          }
+          let itemsToDisable = ['item-1']
+          response.items = response.items.map(item => {
+            if (itemsToDisable.indexOf(item.id) !== -1) {
+              return Object.assign({}, item, {disabled: true})
+            }
+            else {
+              return Object.assign({}, item, {disabled: false})
+            }
+          })
```

Comment:
Tip: This is a good example of where a ternary operator with an implied return in an Array.map makes the code more compact and faster to read.
```
          response.items = response.items.map(item => itemsToDisable.indexOf(item.id) !== -1
              ? { ...item, ...{disabled: true} }
              : { ...item, ...{disabled: false} }
            )
```